google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -193,7 +193,7 @@ clouddebug.errorset=The snapshot location could not be set.
 clouddebug.continuesession=Continue Session
 clouddebug.continueanyway=Continue Anyway
 clouddebug.text=Google Stackdriver Debug
-clouddebug.stackdriver.debugger=Stackdriver Debugger
+clouddebug.stackdriver.debugger=Google Stackdriver Debugger
```

Comment:
Only one place, CloudDebugConfigType.getConfigurationTypeDescription().

Although the Javadoc of this method in ConfigurationType mentions we might want to us the same text as the display name of the configuration type, which is "Google Stackdriver Debug" (clouddebug.text), to keep in line with the existing convention, i.e., call it a Debug run config, not a Debugger run config.

I can do a quick PR into master after we decide: do we want to make getConfigurationTypeDescription() return the same thing as getDisplayName()? I think we should. In that case, clouddebug.stackdriver.debugger will solely be used by the facet.